### ISSUE TITLE
Latest macOS icon style

### ISSUE THREAD

**Issue by okc0mputer** [eccentric-j/doom-icon#3]
How about adding a square icon with rounded corners to match big sur dock icons?

**Comment by jaidetree** [comment]
Oh yes! There's a quick idea I can try, if that doesn't work I'll create an alternate design but that might take a bit longer.

**Comment by okc0mputer** [comment]
Awesome! Let me know if I can help

**Comment by shaunsingh** [comment]
if you look in the emacs-plus repo, I believe theres a few options already, although there are in `.icns` format

**Comment by okc0mputer** [comment]
Thanks, but I’m already using emacs-plus, but none of the doom icons are in the newer square style for macOS

**Comment by shaunsingh** [comment]
Yeah, just deleted my message, didn't realize I was posting in this repo. macosicons.com has the templete, From there its just a matter of inserting the image and exporting

**Comment by pangdaxing23** [comment]
macos Tahoe now unfortunately forces all icons into squircles on a gray background if they're not the new format, making this one stand out a bit. As such, a squircle version would be appreciated. As a comparison:

<img width="236" height="112" alt="Image" src="https://github.com/user-attachments/assets/bdf2f729-395b-4cb5-9d2a-86496f4cce47" />
